trk/src/Screens/SetDetail/index.js

```diff
@@ -55,33 +59,79 @@ function SetDetail(props) {
       });
   }, []);
 
+
+
+  useEffect(() => {
+    const fetchComments = async () => {
+      try {
+        const querySnapshot = await getCommentsBySomeField('climb', climbData.id);
+        const comments = [];
+        querySnapshot.forEach(doc => {
+          const commentData = doc.data();
+          comments.push({
+            id: doc.id,
+            explanation: commentData.explanation,
+            rating: commentData.rating  // Extracting the rating field
+          });
+        });
+        setFeedbackList(comments);
+      } catch (error) {
+        console.error('Error fetching comments:', error);
+      }
+    };
+    fetchComments();
+  }, [climbData.id]);
+
+  renderRating = (rating) => {
+    return '⭐️'.repeat(rating);
+  }
+
+
   return (
-    <View style={styles.container}>
-    <View style={[styles.wrapper]} >
-      <SafeAreaView />
-      <View style={styles.top}>
-        <View style={styles.topLeft}>
-          <View style={styles.gradeCircle}>
-            <Text style={styles.grade}>{climbData.grade}</Text>
+    <ScrollView>
```

Comment:
- Placed the returned view inside a scrollview to deal with feedback that doesnt fit on screen.